docs/releases/v1.24.7.md

```diff
@@ -0,0 +1,54 @@
+# On Premises add-on module release 1.24.7
+
+Welcome to the latest release of `on-premises` module of [`Kubernetes Fury Distribution`](https://github.com/sighupio/fury-distribution) maintained by SIGHUP team.
+
+This minor release adds the compatibility with Kubernetes 1.24.7.
+
+## Package Versions 🚢
+
+| Package                                              | Supported Version | Previous Version  |
+|------------------------------------------------------|-------------------|-------------------|
+| [vmware-cm](katalog/vmware-cm)                       | `1.24.2`          | `Updated`         |
```

Comment:
Shouldn't you add the previous version under the "Previous Version" column?

Also, I would add a link to the upstream project changelog in the "Supported Version" column.

Reply:
You are right, let me change that